Src/GoogleApis.Auth.DotNet4/OAuth2/GoogleWebAuthorizationBroker.cs

```diff
@@ -113,17 +113,22 @@ public class GoogleWebAuthorizationBroker
         /// <param name="taskCancellationToken">Cancellation token to cancel an operation.</param>
         /// <param name="dataStore">The data store, if not specified a file data store will be used.</param>
         /// <returns>User credential.</returns>
-        private static async Task<UserCredential> AuthorizeAsyncCore(
+        public static async Task<UserCredential> AuthorizeAsyncCore(

```

Comment:
The function of ```AuthorizeAsyncCore()``` should already be public in the latest commit unless I'm missing something, I'll double check that it is. I agree that it's neater, would you prefer to force them to use this method specifically by removing the ```includeGrantedScopes``` parameter from the ```AuthorizeAsync()```? That way they'd have no choice but to use the -Core method.